Tests/AtomicsTests/ReferenceTests.swift

```diff
@@ -22,6 +22,47 @@ private class Thing
 
 public class ReferenceTests: XCTestCase
 {
+    
+    public func testRetainCount() {
+        let thing1 = Thing(UInt.randomPositive())
+        let thing2 = Thing(UInt.randomPositive())
+        let originalRetainCount = CFGetRetainCount(thing1)
```

Comment:
It looks like CFGetRetainCount doesn't exist on Linux, so that test fails to compile